src/main/java/hellfall/visualores/mixins/xaerominimap/MinimapRendererMixin.java

```diff
@@ -55,27 +55,27 @@ public abstract class MinimapRendererMixin {
     }
 
     @Redirect(method = "renderMinimap",
-            at = @At(value = "INVOKE", target = "Lxaero/common/minimap/element/render/over/MinimapElementOverMapRendererHandler;render(Lnet/minecraft/entity/Entity;Lnet/minecraft/entity/player/EntityPlayer;DDDDDDDZFLnet/minecraft/client/shader/Framebuffer;Lxaero/common/IXaeroMinimap;Lxaero/common/minimap/render/MinimapRendererHelper;Lnet/minecraft/client/gui/FontRenderer;Lnet/minecraft/client/gui/ScaledResolution;IIIIZF)D")
+            at = @At(value = "INVOKE", target = "Lxaero/hud/minimap/element/render/over/MinimapElementOverMapRendererHandler;render(Lnet/minecraft/util/math/Vec3d;FLnet/minecraft/client/shader/Framebuffer;Lnet/minecraft/client/gui/ScaledResolution;DI)D")
     )
-    private double visualores$injectRender(MinimapElementOverMapRendererHandler instance, Entity renderEntity, EntityPlayer player, double renderX, double renderY, double renderZ, double dimdiv, double ps, double pc, double zoom, boolean cave, float partialTicks, Framebuffer framebuffer, IXaeroMinimap modMain, MinimapRendererHelper helper, FontRenderer font, ScaledResolution scaledRes, int specW, int specH, int halfViewW, int halfViewH, boolean circle, float minimapScale) {
+    private double visualores$injectRender(xaero.hud.minimap.element.render.over.MinimapElementOverMapRendererHandler instance, Vec3d renderPos, float partialTick, Framebuffer framebuffer, ScaledResolution scaledResolution, double backgroundCoordinateScale, int mapDimension) {
```

Comment:
Can you change this to be imported?
```suggestion
    private double visualores$injectRender(MinimapElementOverMapRendererHandler instance, Vec3d renderPos, float partialTick, Framebuffer framebuffer, ScaledResolution scaledResolution, double backgroundCoordinateScale, int mapDimension) {
```